contribute/index.rst

```diff
@@ -1,20 +1,22 @@
 Contributing to UBports
 =======================
 
-Welcome! You're probably here because you want to contribute to UBports. The pages you'll find below here will help you do this in a way that's helpful to both the project and yourself.
-
-If you're just getting started, we always need help with :doc:`thorough bug reporting <bugreporting>`. If you are multilingual, :doc:`translations <translations>` are also a great place to start.
-
-If those aren't enough for you, see `our contribute page <https://ubports.com/page/vo-get-involved>`_ for an introduction of our focus groups.
+Welcome! You're probably here because you want to contribute to UBports.
+The pages below show the ways to contribute that are helpful to both the project and yourself.
```

Comment:
Do we want to provide them in the "easiest first" order (monetary as exception, or first position?)?

Then I would suggest:
1. translation (since on weblate that does not require git knowledge, but more or less equal to bugreporting)
2. bugreporting
3. documentation (not sure if this is more difficult that QA, for doc you need to know what to write, for QA you may need to install stuff on your phone, but maybe not relevant too much anyway)
3. quality-assurance
4. code
5. ppa maintainance
5. monetary

I am not sure about bug-tracking, branchnaming and ppa. 
- To me the headline "ppa" does not sound like a way to contribute. 
- Bug-tracking seems to be more of a description how 2. works, not a separate way to contribute.
- branchnaming is more a description of things you need to know for code contributions

"ppa" could be renamed to "ppa maintainance"

Bug-tracking could be renamed to "issue management". Although that does require special privileges, that is workload and therefore a way to contribute. I would then add it as no. 3, moving the others one down.
We could then change the first paragraph like this:
"This document describes the standard process of dealing with new issues in UBports projects. (Not to be confused with the :doc:`guide on writing a good bugreport </contribute/bugreporting>`.) With the right privileges you could help with triage and issue management."

Also is it "UBports projects"? Or better "Ubuntu Touch projects"? [ppa page, bug tracking page,?]

Branch-naming could be placed as another section under code. I am not happy with this, since the text there is already long. But that is my best idea atm.

Reply:
Ok, I've taken a stab at this ... see what you think. couple selective remarks below

> Do we want to provide them in the "easiest first" order (monetary as exception, or first position?)?

Actually, I don't quite get what a "first position" order is?! But I've tried to follow your proposal a bit closer. Note that some sections moved a level down though. Also, I didn't want to move bug reporting and tracking too far away from each other, so I placed docs below reporting&QA. QA contains tracking now.

>     * To me the headline "ppa" does not sound like a way to contribute.

there isn't an actual headline ppa. it's just the filename. the heading was "repo.ubports.com" now it's "repo names"

>     * Bug-tracking seems to be more of a description how 2. works, not a separate way to contribute.

mhm. really? I don't see bug tracking to be a description of bug reporting .... not how it's written right now and I don't think it should either. I quite like the careful and separate description of both. I go back and forth in my mind about "oh, but tracking requires priviledges, so it's an in-person". But then I shake myself and remind myself that NOT distinguishing too much between insiders and outsiders is a good thing!

>     * branchnaming is more a description of things you need to know for code contributions

moved it underneath code


> With the right privileges you could help with triage and issue management."

again with the insider/outsider. i need to remind myself, but then I prefer to not distinguish it

> Also is it "UBports projects"? Or better "Ubuntu Touch projects"?

beats me. I got no real opinion. I oh so slightly lean towards UBports that's what the orgs in github/gitlab are called ... but yeah ... let me know if you feel strongly and want it changed, no objection

Reply:
> Actually, I don't quite get what a "first position" order is?! But I've tried to follow your proposal a bit closer. Note that some sections moved a level down though. Also, I didn't want to move bug reporting and tracking too far away from each other, so I placed docs below reporting&QA. QA contains tracking now.

Sorry, that was not clear enough phrased. Sorting after what would be the easiest things a contributor can do/contribute. And the "first position" referred only to monetary support. That is at the bottom, but **could** be placed at the first position because giving money might be the easiest thing to do.

Reply:
> there isn't an actual headline ppa. it's just the filename. the heading was "repo.ubports.com" now it's "repo names"

Sorry. I mixed that up. :-)

Reply:
I very much like the order you gave it now. Well done!

Not sure if the naming conventions are a bit too far down. They could be under "Code change checklist" as a fourth headline "Follow our naming conventions". It does look a bit lost there at the end. On the other hand more of a minor thing. Might also be just my personal view on a Friday evening. ;-)

Reply:
> beats me. I got no real opinion. I oh so slightly lean towards UBports that's what the orgs in github/gitlab are called ... but yeah ... let me know if you feel strongly and want it changed, no objection

Well, as far as I understand (correct me if I am wrong), Ubuntu Touch is the OS developed by the community. UBports is a foundation that supports the community. So I thought the projects are for the OS=Ubuntu Touch, and UBports supports the development needed to finish the projects. That was when I wrote that review. :-)

Checking now the naming, [https://github.com/orgs/ubports/projects](https://github.com/orgs/ubports/projects) or [https://gitlab.com/ubports](https://gitlab.com/ubports), it is all UBports. 

So I guess we can/should leave it as it is. It might need someone who set that all up to provide some final insight on this.